src/web/error.rs

```diff
@@ -117,6 +117,11 @@ impl From<Context<ValidationErrorKind>> for ValidationError {
                     _ => StatusCode::BAD_REQUEST,
                 }
             }
+            ValidationErrorKind::FromDetails(ref _description, ref location, Some(ref _name))
+                if *location == RequestErrorLocation::Path =>
```

Comment:
Same here (too broad) but I think this one can be further restricted to && name == "bso" || name == "collection"

Otherwise it hits another ::Path case (triggered by the HawkIdentifier)

Reply:
Well, there's no name, but I agree that I can safely limit to location == body.

Reply:
The `Some(ref _name)` I meant